packages/grid/x-grid-modules/src/hooks/features/useSorting.ts

```diff
@@ -170,22 +164,22 @@ export const useSorting = (
       order.push(row.id);
       return order;
     }, [] as RowId[]);
-  }, [originalOrder, apiRef]);
```

Comment:
Better not include the ref in the list of the dependencies. It's not needed and simpler without.